pages/osx/setquota.md

```diff
@@ -0,0 +1,24 @@
+# setquota
```

Comment:
Mandatory fix: [specification violation](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform): this page can be safely removed: it's the same as `linux/setquota.md`.

> If a page is common across multiple platforms, but slightly differently on a given platform, then the page is still stored in common, but a copy tailored for the differing platform is placed in that platform's specific folder.